docs/source/conf.py

```diff
@@ -138,9 +138,9 @@
     "goatcounter_url": "https://docs-conda-io.goatcounter.com/count",
     "icon_links": [
         {
-            "name": "Element",
-            "url": "https://matrix.to/#/#conda:matrix.org",
-            "icon": "_static/element_logo.svg",
```

Comment:
I couldn't find the element_logo.svg file to delete it. Is this rolled up in the theme?

Reply:
The changout of the Zulip icon might be a separate task, if so.